docs/guides/continuous-integration/circleci.mdx

```diff
@@ -35,11 +37,11 @@ A typical project can have:
 ```yaml
 version: 2.1
 orbs:
-  # "cypress-io/cypress@2" installs the latest published
+  # "cypress-io/cypress@3" installs the latest published
   # version "s.x.y" of the orb. We recommend you then use
-  # the strict explicit version "cypress-io/cypress@2.x.y"
+  # the strict explicit version "cypress-io/cypress@3.x.y"
   # to lock the version and prevent unexpected CI changes
-  cypress: cypress-io/cypress@2
+  cypress: cypress-io/cypress@3
 workflows:
   build:
     jobs:
```

Comment:
A nice addition could be add a line after this code snippet like, 

"That's it! Your repo's dependencies will be installed and cached and your Cypress tests will run in CircleCI" 

To try to bring home the simplicity offered here.

Reply:
Resolved in aade935bf6e28eb47483f83d95143e6d3dd8286d